lib/facter/easyrsa.rb

```diff
@@ -9,12 +9,12 @@
     operatingsystemrelease = Facter.value(:operatingsystemrelease)
 
     case operatingsystem
-    when %r{RedHat|CentOS|Amazon}
+    when %r{RedHat|CentOS|Amazon|Rocky|AlmaLinux|OracleLinux}
       binaryv2 = '/usr/share/easy-rsa/2.0/pkitool'
       binaryv3 = '/usr/share/easy-rsa/3/easyrsa'
     when %r{Ubuntu|Debian}
       case operatingsystemrelease
-      when %r{9|10|11|18.04|20.04|22.04}
+      when %r{9|10|11|12|18.04|20.04|22.04|24.04}
```

Comment:
Isn't this effectively all supported versions? In other words, perhaps just drop the whole `case operatingsystemrelease` here.

Reply:
I just forgot to remove the old one, remove asap